pages/settings/general/index.tsx

```diff
@@ -15,7 +15,7 @@ import {
 import { updateSettings, fetchSettings } from '@/ducks/settings';
 
 import View from './view';
-import _ from 'lodash';
+import { some, isEqual } from 'lodash';
```

Comment:
See **Takeaways** seconds point please: https://www.azavea.com/blog/2019/03/07/lessons-on-tree-shaking-lodash/

```suggestion
import some from 'lodash/some';
import isEqual from 'lodash/isEqual';
```